string Name
string Topic
time StartTime
duration ExecutionTime
---
time RegisteredStartTime